src/main/java/eu/darkbot/api/PluginAPI.java

```diff
@@ -1,10 +1,17 @@
 package eu.darkbot.api;
 
+import eu.darkbot.api.plugin.Module;
+import eu.darkbot.api.utils.Version;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 public interface PluginAPI {
 
+    /**
+     * {@link Version} of the DarkBot.
+     */
+    Version VERSION = Version.of("1.13.17 beta 59");
```

Comment:
probably doesn't belong here, it's not the version of the plugin api, it's the version of the bot